xds/src/main/java/io/grpc/xds/internal/sds/XdsChannelBuilder.java

```diff
@@ -32,7 +30,6 @@
  * A version of {@link ManagedChannelBuilder} to create xDS managed channels that will use SDS to
  * set up SSL with peers. Note, this is not ready to use yet.
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/6268")
```

Comment:
Why was this removed? Issue 6268 would still be open until we have implemented XdsCredentials API

Reply:
Because it's moved to internal. `@ExperimentalApi` is for APIs that are not internal.

Reply:
We can close #6268 and open another once we have to add a public API in `io.grpc.xds`.